x-pack/plugins/security_solution/server/lib/detection_engine/rules/bulk_actions/rule_params_modifier.test.ts

```diff
@@ -351,4 +351,28 @@ describe('ruleParamsModifier', () => {
       expect(editedRuleParams.timelineTitle).toBe('Test timeline');
     });
   });
+
+  describe('schedule', () => {
+    test('should set timeline', () => {
+      const INTERVAL_IN_MINUTES = 5;
+      const LOOKBACK_IN_MINUTES = 1;
+      const FROM_IN_SECONDS = (INTERVAL_IN_MINUTES + LOOKBACK_IN_MINUTES) * 60;
+      const editedRuleParams = ruleParamsModifier(ruleParamsMock, [
+        {
+          type: BulkActionEditType.set_schedule,
+          value: {
+            interval: `${INTERVAL_IN_MINUTES}m`,
+            lookback: `${LOOKBACK_IN_MINUTES}m`,
+          },
+        },
+      ]);
+
+      // @ts-expect-error
+      expect(editedRuleParams.interval).toBeUndefined();
```

Comment:
Can it be fixed?

Reply:
No, in this case the error is actually expected. `interval` should not exist in `editedRuleParams`; just asserting that, to be a 100% sure that the separation between params and attributes is working as expected.

Reply:
@jpdjere `@ts-expect-error` cover all the possible errors which can be dangerous. This could be a safer option:

```
      // eslint-disable-next-line @typescript-eslint/no-explicit-any
      expect((editedRuleParams as any).interval).toBeUndefined();
```

Reply:
Ok, will add to: https://github.com/elastic/kibana/pull/141604